### PR TITLE
Adds admin version of user profile

### LINKED ISSUE #54: User Story 20, Admin User Profile Page
```
As an admin user
When I visit a user's profile page ("/admin/users/5")
I see the same information the user would see themselves
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #58

child of #79

depends on #73

depends on #3

### PR DESCRIPTION
**Pull request by timnallen**
As an admin user
When I visit a user's profile page ("/admin/users/5")
I see the same information the user would see themselves

Closes #54

### CODE REVIEW

**Review by allieIsOkay — APPROVED** [review]
Looks good tim!

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,12 +1,7 @@
 <section>
   <h1>Your profile</h1>
   <article class="user-info">
-    <p>Username: <%= current_user.name %> </p>
-    <p>Email: <%= current_user.email %> </p>
-    <p>Address: <%= current_user.address %> </p>
-    <p>City: <%=  current_user.city %> </p>
-    <p>State: <%= current_user.state %> </p>
-    <p>Zip code: <%= current_user.zipcode %> </p>
+    <%= render partial: "users/profile" %>
```
Comment (csvlewis) [inline app/views/users/show.html.erb:4]: Good refactor, Tim! I like the use of partials

**Review by csvlewis — APPROVED** [review]

File: app/views/users/show.html.erb (hunk 1 above)
Comment (teresa-m-knowles) [reply app/views/users/show.html.erb:4]: I agree!

**Review by teresa-m-knowles — APPROVED** [review]
This looks really good, @timnallen
